### DISCUSSION TITLE
有没有望支持以下功能

### DISCUSSION THREAD

**Discussion by xlingzzy** [lollipopkit/flutter_server_box#24] [Ideas]
ssh连接后的那个背景

在线编辑配置文件

希望能尽量支持的是第一个 因为看着这白色背景好单调

**Comment by PaperCube** [comment]
“ssh连接后的那个背景”，是指背景颜色和前景颜色吗

**Reply by xlingzzy** [reply]
![Screenshot_2023-04-03-14-25-37-627_tech.lolli.toolbox.jpg](https://user-images.githubusercontent.com/88002557/229427993-72993ed6-338c-4d4a-a5bc-6f073e6c6aed.jpg)

我也不知道咋说 就这个背景 白色的太单调了  想弄成背景图

**Comment by xlingzzy** [comment]
然后那个ssh好像有点问题 断线了咱也不知道 呜呜

**Comment by ghost** [comment]
> ssh连接后的那个背景
> 
> 在线编辑配置文件
> 
> 希望能尽量支持的是第一个 因为看着这白色背景好单调
> 
> 我也特别希望加入能够编辑文件，现在ssh一个，sftp编辑文件一个多少有点麻烦
